src/js/views/patients/sidebar/filters/filters-sidebar.scss

```diff
@@ -1,6 +1,12 @@
 .filters-sidebar__clear-filters {
   color: color(blue);
   font-weight: 600;
+  margin-right: 16px;
```

Comment:
Removed the `u-margin--r-16` class name from this element and applied it here instead. Seemed unnecessary if we already have a class name added to that element.